src/server/handlers/github.js

```diff
@@ -137,3 +149,35 @@ const getRequest = (owner, name, token, secret) => {
     }
   };
 };
+
+/*
+ * parse_link_header()
+ *
+ * Parse the Github Link HTTP header used for pageination
+ * http://developer.github.com/v3/#pagination
+ *
+ * Modified by kfenn to return page numbers instead of urls
```

Comment:
@cjwatson: Yeah, extracting page numbers from the GitHub API response still saves us work, which I am trying to do at every opportunity.